doc/guide/feature-docker.xml

```diff
@@ -28,4 +28,6 @@
 $ <command>sudo docker run -ti fedora /bin/bash</command>
 [root@57625bc8787e /]#
 </programlisting>
+
+  <para>Please note that <code>docker</code> support has been deprecated since <ulink url="https://cockpit-project.org/blog/cockpit-215.html">version 215</ulink> in favor of <code>cockpit-podman</code>, and will not be maintained upstream anymore.</para>
```

Comment:
Thank you!
I think it would be better on top of the page. And can you please also link `cockpit-podman` to https://github.com/cockpit-project/cockpit-podman/

It would be cool if we could make it look like warning, like for example [here](https://cockpit-project.org/blog/creating-plugins-for-the-cockpit-user-interface.html). [source for that](https://github.com/cockpit-project/cockpit-project.github.io/blob/master/_posts/2014-11-13-cockpit-plugin-tutorial.md). Not sure though if possible, @garrett might know.

Reply:
Where do you want the warning?

If on the website, I added that in https://github.com/cockpit-project/cockpit-project.github.io/pull/320 — that PR includes the support and several changes. (It's just `{:.warning}` on a new line right after the warning in Markdown.)

Otherwise, I think `<p class="warning">This is a warning</p>` is enough for HTML?

I'm not sure how you'd add a class to the XML here.

Reply:
ah right, then no fancy warning :)

Reply:
Why not? It's as simple as adding a "warning" class.

Reply:
Oh, https://cockpit-project.org/guide/latest/embedding has a warning in the guide. There's some inline CSS that's odd (and I guess I should override in that case), but it otherwise works.

Reply:
Here you go: https://github.com/cockpit-project/cockpit/blob/cb29836c3c048f024e7062b10585a9b7f1abf99f/doc/guide/embedding.xml#L78-L80

Reply:
I'll work on fixing the CSS for the website.

Reply:
Oh I completely misunderstood. So yeah @sogalgeeko can you please use that? Thanks!

Reply:
Fix for CSS when this PR for the layout lands: https://github.com/cockpit-project/cockpit-project.github.io/pull/342 — but it'll probably "just work" even without the fix, as most warnings in documentation (and on the website) do.

Reply:
> Oh I completely misunderstood. So yeah @sogalgeeko can you please use that? Thanks!

Will do!

Reply:
@marusak @garrett Done in https://github.com/cockpit-project/cockpit/pull/14490/commits/b79c582f1860a05cdd2acedb0666763537412244
I hope it's OK.

Reply:
Looks good in the XML :+1:; I hope it works as intended once this and the CSS change both land.

Reply:
Could you please also address my other two comments? Thanks!

Reply:
@marusak My bad, I omitted it, fixed in https://github.com/cockpit-project/cockpit/pull/14490/commits/61f57d375249486f0d943c2afa435ce1d0762796